ls1intum/Artemis
# [Followup] fix problems with creating example solutions and bugs in TextAssessmentResource

### Checklist
- [x] Server: I implemented the changes with a good performance and prevented too many database 

### Motivation and Context



follow-up for the refactoring of the Result- and TextAssessmentResource

### Description

added Tests

### Steps for Testing

check the console for errors such as badRequest related to the Endpoints in TextAssessmentResource and ResultResource while you:
1. create a text exercise
2. participate in it twice
3. assess one participation
4. start assessing the second one, dont submit and go back to the assessment dashboard, then continue assessing the exercise
5. create a n example submission with assessment and save it
6. assess a text exercises in an exam (maybe use the test run feature here, makes it faster to test instead of conducting a whole 
7.  exam ^^)

code review for test additions

### Test Coverage




ResultResource: 93%
TextAssessmentResource: 91%
